unit2-java/lesson14.rst

```diff
@@ -7,8 +7,9 @@ Goals
 Objectives
 ----------
 
-- Utilize *phpMyAdmin* and build a database locally
 - Create a new user and database
-- Import tables using a `.sql` file
 - Explain how and why data is split into related tables
+- Utilize foreign keys
+- Create tables with an understanding of one-to-many relationships
 - Perform complex SQL queries with joins and subqueries
+
```

Comment:
How about two additional objectives to round out the SQL lessons?

1. Import tables using a `.sql` file and *MySQL Workbench*
2. Write simple queries using LEFT, RIGHT, and FULL joins.